types/draft.ts

```diff
@@ -1,13 +1,19 @@
 import { z } from 'zod'
 
-const Draft = z.object({
-	title: z.string().max(48).optional(),
-	summary: z.string().max(255).optional(),
-	description: z.string().max(2048).optional(),
-	type: z.enum(['topic', 'project']).optional(),
+const DraftResponse = z.object({
```

Comment:
@randomontherun question: I'd like to look at this / talk through it with you. It seems like a smell to me that we need these to be both optional and nullable rather than something simpler like always returning a string, just an empty one if we don't have another value for it.